cypress/utils/utils.ts

```diff
@@ -1373,8 +1373,8 @@ export function goToPage(page: number): void {
 }
 
 export function selectUserPerspective(userType: string): void {
-    cy.get(commonView.optionMenu).click();
-    cy.get(commonView.actionMenuItem).contains(userType).click();
+    cy.get("button.pf-v5-c-menu-toggle.pf-m-full-width").click({ force: true });
```

Comment:
Hi @sshveta , The selector hasn't changed . I'm using a different locator to get the job done because with the current locator, there are intermittent issues and the user perspective toggle doesn't always get clicked. 

Sure, I could run the PR on additional tests .

Reply:
This PR tests the following -
1) Creation and deletion of applications - 'Migration/Application inventory'
2) Creation and deletion of stakeholders - 'Migration/Controls/Stakeholders'
3) Creation and deletion of questionnaires - 'Migration/Administration/Assessment questionnaires'